lib/ansible/plugins/lookup/conjur_variable.py

```diff
@@ -0,0 +1,155 @@
+# (c) 2018, Jason Vanderhoof <jason.vanderhoof@cyberark.com>, Oren Ben Meir <oren.benmeir@cyberark.com>
+# (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: conjur_variable
+    version_added: "2.5"
+    short_description: Fetch credentials from CyberArk Conjur.
+    description:
+      - Retrieves credentials from Conjur using the server's Conjur identity. More information about Conjur can be found at https://www.conjur.org/.
+    requirements:
+      - The machine running Ansible has been given a Conjur identity.
```

Comment:
I assume this process is obvious to process to people using this plugin, and therefore doesn't need further explanation?

Reply:
This will be familiar to Conjur users but I'll add a link to documentation to help people who are learning Conjur and Ansible at the same time.